resources/tests/conftest.py

```diff
@@ -64,14 +64,18 @@ def winter_storage_place():
 
 
 @pytest.fixture
-def user():
-    user = UserFactory()
-    return user
-
+def user(request):
```

Comment:
could you please move this fixture to `berth_reservations/tests/conftest.py` as it is probably gonna be used by multiple apps within the project 🙂 this is also why I was asking about default `request.param`, since other tests might use it directly

Reply:
Sure! I can also default no param to the base user (I think it would make more sense)